config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -141,17 +151,17 @@ spec:
                               type: integer
                             type: array
                           interval:
-                            minimum: 0
```

Comment:
removing this minimum would technically allow negative numbers, is that something we can add back with a directive?

Goes for all other `minimum: 0` declarations in this diff as well.